tests/suites/1_stateful/00_copy/00_0011_dirtributed_copy_from_stage_for_text_file.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+. "$CURDIR"/../../../shell_env.sh
+
+echo "drop table if exists products;" | $MYSQL_CLIENT_CONNECT
```

Comment:
BTW, how can we know the COPY is in distrubuted? Can we EXPLAIN it?

Reply:
explain doesn't not work well. we can't make sure the distributed copy from explain